systemd/starlink-mqtt.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Starlink GRPC to MQTT exporter
+After=network.target
+
+[Service]
+Type=simple
+WorkingDirectory=/opt/starlink-grpc-tools/
+Environment=MQTT_HOST=localhost MQTT_PORT=1883 MQTT_USERNAME=<changeme> MQTT_PASSWORD=<changeme> MQTT_SSL=false
```

Comment:
I see you copied the `MQTT_SSL=false` bit from the influxdb2 systemd script, but that's actually broken in that script because the env variable processing will treat it as setting the CA cert file to the string "false". Whether SSL is enabled in the influxdb2 script is based on the URL, though, so that won't actually break anything.

I guess there are 2 options here. The env var processing can be (further) special cased to ignore "false", thus leaving the default of TLS disabled, or that bit can be removed from the example here.

Reply:
I'm not using TLS on my MQTT so I just went on the assumption that it worked.  For completeness, it probably should support TLS if the options are specified.

Reply:
Ha! I just assumed it worked, too, when I accepted the PR for the inxluxdb2 script.

No worries. I'll fix it so that "false" is a no-op when I fix the TLS options.